src/backup-metadata/test/e2e/e2e_test.go

```diff
@@ -0,0 +1,59 @@
+package e2e_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"os/exec"
+	"strconv"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"code.cloudfoundry.org/capi-k8s-release/src/backup-metadata/internal/cfmetadata"
+)
+
+var _ = Describe("e2e", func() {
+	It("Assert CF metadata structure", func() {
+		backupName := generateRandomName()
+		createBackup(backupName)
+		cfMetadataJSON := getCFMetadataFromBackupLogs(backupName)
+
+		var metadata cfmetadata.Metadata
+		err := json.Unmarshal([]byte(cfMetadataJSON), &metadata)
+		Expect(err).NotTo(HaveOccurred())
+
+		Expect(metadata.Totals.Orgs).Should(BeNumerically(">=", 1))
+		Expect(metadata.Totals.Spaces).Should(BeNumerically(">=", 0))
+		Expect(metadata.Totals.Users).Should(BeNumerically(">=", 0))
+		Expect(metadata.Totals.Apps).Should(BeNumerically(">=", 0))
```

Comment:
Testing app/user/space backup would also be nice in this e2e test. Happy to help set that up if desired -- it'd be pretty straightforward to do with a few CF CLI commands.